specification/recoveryservicessiterecovery/resource-manager/readme.md

```diff
@@ -45,6 +45,11 @@ directive:
       - R4011
     reason: service implements 204 for delete and DeleteOperationResponses error was falsely raised.
 
+  - from: service.json
+    suppress:
```

Comment:
>suppress [](http://example.com/codeflow?start=4&length=8)

please use where clause and mention the paths, instead of suppressing for whole file.

Reply:
Updated